apps/frontend/next.config.js

```diff
@@ -2,13 +2,28 @@ const { withSentryConfig } = require('@sentry/nextjs')
 const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true'
 })
-
+require('dotenv').config({ path: '../../.env' })
 /** @type {import('next').NextConfig} */
+const APP_ENV = process.env.APP_ENV //|| 'local'
+const MEDIA_BUCKET_NAME = process.env.MEDIA_BUCKET_NAME
+const STORAGE_BUCKET_ENDPOINT_URL = process.env.STORAGE_BUCKET_ENDPOINT_URL
+
+const domains =
+  APP_ENV === 'production'
+    ? [`${MEDIA_BUCKET_NAME}.s3.ap-northeast-2.amazonaws.com`]
+    : APP_ENV === 'stage'
+      ? ['stage.codedang.com']
+      : ['stage.codedang.com']
```

Comment:
```suggestion
 process.env.NODE_ENV === 'production'
    ? [`${MEDIA_BUCKET_NAME}.s3.ap-northeast-2.amazonaws.com`]
    : ['stage.codedang.com']
```

이건 어떤가요?? 동일하게 동작할 것 같아요!

Reply:
추가된 환경변수가 있다면 아래 문서 업데이트해주세요!

https://www.notion.so/skkuding/env-bf3363643b144dd48b59063e4d7e9664